docs/platforms/android/session-replay/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Set Up Beta Session Replay for Mobile
+sidebar_order: 5500
+notSupported:
+description: "Learn how to enable the Beta of Mobile Session Replay in your app."
+---
+
+<Note>
+
+Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+
+If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-java/issues/new?assignees=&labels=Platform%3A+Android%2CType%3A+Bug&projects=&template=bug_report_android.yml) with a link to a relevant replay in Sentry if possible.
+
+</Note>
+
+[Session Replay](/product/explore/session-replay/) helps you get to the root cause of an error or latency issue faster by providing you with a reproduction of what was happening in the user's device before, during, and after the issue. You can rewind and replay your application's state and see key user interactions, like taps, swipes, network requests, and console entries, in a single UI.
+
+By default, our Session Replay SDK masks all text content, images, and user input, giving you heightened confidence that no sensitive data will leave the device. To learn more, see <PlatformLink to="/product/session-replay/">Session Replay Product docs</PlatformLink>.
+
+## Pre-requisites
+
+Make sure your Sentry Android SDK version is at least 7.12.0.
+
+## Install
+
+The easiest way to update through the Sentry Android Gradle plugin to your app module's `build.gradle` file.
+
+```groovy {filename:app/build.gradle}
+plugins {
+  id "com.android.application"
+  id "io.sentry.android.gradle" version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
+}
+```
+
+```kotlin {filename:app/build.gradle.kts}
+plugins {
+  id("com.android.application")
+  id("io.sentry.android.gradle") version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
+}
+```
+
+If you have the SDK installed without the Sentry Gradle Plugin, you can update the version directly in the `build.gradle` through:
+```groovy {filename:app/build.gradle}
+dependencies {
+    implementation 'io.sentry:sentry-android:{{@inject packages.version('sentry.java.android', '7.11.0') }}'
+}
+```
+
+```kotlin {filename:app/build.gradle.kts}
+dependencies {
+    implementation("io.sentry:sentry-android:{{@inject packages.version('sentry.java.android', '7.11.0') }}")
+}
+```
+
```

Comment:
In theory, we could also add a snippet on how to remove session replay if it's not used (it's a separate package) - do you think we should do it, or leave for something like Troubleshooting page?

Reply:
We usually don't add this as it's the inverse of adding. We could add if it makes sense, like you mean not having `sentry-android` and going with `sentry-android-core` ?

Definitely a GA level doc we'd want to have at least then

Reply:
yeah, let's skip it for now, I think it's for GA or Troubleshooting. Yeah, I meant not having that code packaged at all, if it's not used.